vendor/puppeteer-core/vendor/zip/mod.ts

```diff
@@ -1,6 +1,6 @@
 import _JSZip from "https://dev.jspm.io/jszip@3.5.0";
-import { walk, WalkOptions } from "https://deno.land/std@0.93.0/fs/walk.ts";
-import { join, SEP } from "https://deno.land/std@0.93.0/path/mod.ts";
+import { ensureDir, walk, WalkOptions } from "https://deno.land/std@0.93.0/fs/mod.ts";
```

Comment:
can you import from the individual modules instead of from `mod.ts`? want to keep deps as slim as possible.

Reply:
Sure.